app/helpers/application_helper.rb

```diff
@@ -1214,7 +1215,7 @@ def start_page_allowed?(start_page)
   end
 
   def allowed_filter_db?(db)
-    return false if db.start_with?('Container') && !get_vmdb_config[:product][:containers]
+    return false if (db.nil? || db.start_with?('Container')) && !get_vmdb_config[:product][:containers]
     true
```

Comment:
This is still broken.. do you actually need the `db.nil?` logic? Feels like you're introducing a bug.. Why should it return false for nil, but true for ""? (Assuming `:product` `:containers` being falsy).